app/modules/sightings/models.py

```diff
@@ -41,6 +41,24 @@ def get_owner(self):
             return self.get_owners()[0]
         return None
 
+    # will return None if not a single owner of all encounters (otherwise that user)
```

Comment:
This feels to be in the wrong file. This is the permissions that we are applying to sighting. For project/Org, the logic of how permissions are applied is in the permissions file in the _permitted_via_org and _permitted_via_project functions.
I think this should be another private function in the ObjectActionRule class in permissions

Reply:
i could see the bottom two (`user_*_encoungers()`) being moved -- they are testing against a specific user; thus are pretty permissions-y.  however, i feel like `single_encounter_owner()` could be useful elsewhere off of any arbitrary sighting.   it returns an owner-_like_ object akin to `get_owners()` and `get_owner()` above it.  maybe?

(i would argue that the singular `get_owner()` should just go away; but thats another issue.)